test/driver_test.rb

```diff
@@ -90,12 +90,23 @@
         driver: @driver,
         passenger_id: 3,
         start_time: Time.new(2016, 8, 8),
-        end_time: Time.new(2016, 8, 8),
+        end_time: Time.new(2016, 8, 9),
         rating: 5
       )
       @driver.add_trip(trip)
     end
 
+    let (:trip2) {
+      trip2 = RideShare::Trip.new(
+          id: 8,
+          driver: @driver,
+          passenger_id: 3,
+          start_time: Time.new(2016, 8, 8),
+          end_time: Time.new(2016, 8, 9),
+          rating: 1
+      )
```

Comment:
You don't need to (and shouldn't) assign this to a variable:

```suggestion
      RideShare::Trip.new(
          id: 8,
          driver: @driver,
          passenger_id: 3,
          start_time: Time.new(2016, 8, 8),
          end_time: Time.new(2016, 8, 9),
          rating: 1
      )
```